classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
```

Comment:
***Since you don't have commits for the `student.rb` file, I'll add the comments here.***

In the Student class:
Kindly add a method that serves as the 'belongs-to' side (a student belongs to a classroom).

i.e
```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Also please add `attr_reader` for `classroom` so that it is accessible from the Student object.